packages/jaeger-ui/src/components/SearchTracePage/SearchResults/index.test.js

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2017 Uber Technologies, Inc.
+// Copyright (c) 2023 Uber Technologies, Inc.
```

Comment:
why is this changed?

Reply:
I am wondering if we need to keep the year up to date if we have the code lines in the file get updated?
Or do we keep them as origin?

Reply:
Uber's was the original copyright, new contributions should use "The Jaeger Authors", which you can add as a line above with the current year if you want.

We tend to keep the original year a specific (c) by-line was introduced. I checked with legal once, there wasn't a lot of value in updating the year.

Reply:
Sounds good! I'm going to roll back the corresponding change as it is not necessary. Thx for letting me know:)